packages/dashboard-v2/src/components/forms/LoginForm.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from "prop-types";
+import { Formik, Form } from "formik";
+import { Link } from "gatsby";
+import * as Yup from "yup";
+
+import HighlightedLink from "../HighlightedLink";
+import { TextField } from "../Form/TextField";
+import { Button } from "../Button";
+
+import accountsService from "../../services/accountsService";
+
+const loginSchema = Yup.object().shape({
+  email: Yup.string().required("Email is required").email("Please provide a valid email address"),
+  password: Yup.string().required("Password is required"),
+});
+
+const INVALID_CREDENTIALS_ERRORS = ["password mismatch", "user not found"];
+
+export const LoginForm = ({ onSuccess }) => (
+  <Formik
+    initialValues={{
+      email: "",
+      password: "",
+    }}
+    validationSchema={loginSchema}
+    onSubmit={async (values, { setErrors }) => {
+      try {
+        await accountsService.post("login", {
+          json: values,
+        });
+
+        onSuccess();
+      } catch (err) {
+        if (err.response) {
```

Comment:
we should handle a case where err.response is empty too, that's a timeout for example, we should display a toast or sth like that (can be in follow up)

Reply:
Good point. Here are some examples after fixes:

* [invalid credentials](https://user-images.githubusercontent.com/3853033/159941381-16f6e0b5-04ce-453f-936b-198b1b060b6c.png)
* [connectivity issues - offline, timeout](https://user-images.githubusercontent.com/3853033/159941511-c645ca96-d617-47a7-afbe-2939378e61f7.png)

Reply:
Addressed in 731b1b6d